.github/workflows/snapshot-tests.yml

```
@@ -0,0 +1,269 @@
+    runs-on: ubuntu-22.04
+    outputs:
+      runner: ${{ steps.alt-runner.outputs.runner || 'ubuntu-22.04' }}
+    steps:
+      - id: alt-runner
+        if: inputs.use_buildjet_runner
+        run: echo "runner=buildjet-4vcpu-ubuntu-2204" >> $GITHUB_OUTPUT
+
+  snapshots:
+    name: End-to-end snapshot tests
+    needs: snapshots-instance-choice
+    if: ${{ inputs.network == '' || inputs.network == matrix.network.name }}
```

actionlint (1.7.11)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)